### ISSUE TITLE
In order to give output sound for notifications edit the HTML code at local host.

### ISSUE THREAD

**Issue by adiSarvesh** [jitsi/jitsi-meet#13543]
\

**Comment by saghul** [comment]
I don't understand what you are trying to say. Can you rephrase?

**Comment by adiSarvesh** [comment]
Hey,I am trying to say that in your local host create and upload one file
of audio_notification.mp3 .
Use the code i gave you in order to add notification sound at line 403 of
HTML file.

**Comment by saghul** [comment]
And why would we want to do that?
